css/properties/-webkit-mask-attachment.json

```diff
@@ -0,0 +1,54 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-mask-attachment": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-attachment",
+          "support": {
+            "webview_android": {
+              "version_added": "2.1"
```

Comment:
I would put null here as the value is dubious.

Reply:
Done!

Reply:
Done!